cognite/client/_api/datapoints.py

```diff
@@ -1017,8 +1017,8 @@ def retrieve_dataframe_in_tz(
             aggregates with automatic handling for daylight saving time (DST) transitions. If your time zone observes DST,
             and your query crosses at least one DST-boundary, granularities like "3 days" or "1 week", that used to represent
             fixed durations, no longer do so. To understand why, let's illustrate with an example: A typical time zone
-            that observes DST will skip one hour ahead during spring, leading to a day that is only 23 hours long, and oppositely
-            in the fall, turning back the clock one hour, yielding a 25-hour long day.
+            (above Equator) that observes DST will skip one hour ahead during spring, leading to a day that is only
```

Comment:
Seems capitalizing "equator" isn't strictly wrong, just not recommended. You choose 😉 
```suggestion
            (above the equator) that observes DST will skip one hour ahead during spring, leading to a day that is only
```